vividus-extension-selenium/src/main/java/org/vividus/selenium/manager/WebDriverManagerContext.java

```diff
@@ -16,17 +16,13 @@
 
 package org.vividus.selenium.manager;
 
-import java.util.function.Supplier;
-
-public interface IWebDriverManagerContext
+public interface WebDriverManagerContext<K>
 {
-    <T> T getParameter(WebDriverManagerParameter parameter);
-
-    <T> void putParameter(WebDriverManagerParameter parameter, T value);
+    <T> T get(K key);
 
-    <T> T get(WebDriverManagerParameter parameter, Supplier<T> initialValueSupplier);
+    <T> void put(K key, T value);
 
     void reset();
 
-    void reset(WebDriverManagerParameter parameter);
+    void reset(K key);
```

Comment:
is it still needed?

Reply:
im might be useful if we cache some dynamic data like screen orientation